.jenkins/precheckin-cuda.groovy

```diff
@@ -72,7 +72,7 @@ ci: {
     {
         properties(auxiliary.addCommonProperties([pipelineTriggers([cron('0 1 * * 6')])]))
         stage(urlJobName) {
-            runCI(['ubuntu20-cuda11':['anycuda']], urlJobName)
+            runCI(['ubuntu20-cuda11':['anycuda'],'centos8-cuda11':['anycuda']], urlJobName)
```

Comment:
We might want to just focus our effort on RHEL8. Not sure about the status of that container.